buildenv/travis/build-on-travis.sh

```diff
@@ -30,6 +30,17 @@ set -x
 # builds have 2 cores and 4 gigs of memory.  Attempt to double provision the number of cores for the make
 MAKE_JOBS=4
 
+# download bootstrap jdk
+SRCDIR=$PWD
+cd ~
+wget -O bootjdk10.tar.gz "https://api.adoptopenjdk.net/v2/binary/releases/openjdk10?openjdk_impl=openj9&os=linux&arch=x64&release=latest&type=jdk"
```

Comment:
We can use jdk11 to build jdk11, which is what we use in other jdk11 builds. Since jdk10 is out of service, using jdk11 should be more robust for the future.